picard/tagger.py

```diff
@@ -213,15 +293,16 @@ class Tagger(QtWidgets.QApplication):
     _debug = False
     _no_restore = False
 
-    def __init__(self, picard_args, localedir, autoupdate, pipe_handler=None):
+    def __init__(self, picard_args, localedir, autoupdate, pipe_handler=None, to_load=None):
```

Comment:
Since you've told me to keep such changes away from the feature PR, when this one gets merged, I'll refactor the class to take one arg less by embedding the remaining picard args into `to_load`